app/presenters/buyers/accounts_index_presenter.rb

```diff
@@ -14,7 +14,7 @@ def initialize(provider:, params: {})
 
   def buyers
     @buyers ||= provider.buyer_accounts
-                        .includes([:admin_user])
+                        .includes(%i[admin_users admin_user])
```

Comment:
why is `admin_user` not enough?

Reply:
I think it's enough, and all the first_admin additional logic was redundant, I removed it and the n+1 test passes